msfits/MSFits/FitsIDItoMS.h

```diff
@@ -129,7 +129,9 @@ class FITSIDItoMS1 : public BinaryTableExtension
   // software correlator used by the EVN).
   //
 
-  FITSIDItoMS1(FitsInput& in, const String& correlat, const Int& obsType=0, const Bool& initFirstMain=True);
+  FITSIDItoMS1(FitsInput& in, const String& correlat,
+	       const Float& vanVleck=0.0, const Int& obsType=0,
```

Comment:
When adding a new argument with a default value, I think it's better to put it at the end.

Reply:
I can see your point. Conceptually the new vanVleck parameter is related to the existing correlat parameter, so that is why I added it there.  But I can change it.

This actually relates to what I consider to be a bit of an issue with how the code is structure. I think that for all practical purpose the FITSIDItoMS1 class should be considered an "internal" API. Users should be using the MSFitsIDI class (and this is what importfitsidi does). But there is nothing in the documentation that spells this out.

Reply:
Given that there is no documentation (either about the parameters, or about the fact that this should be an internal API), could you move the parameter to the end to avoid bugs?